client/me/account-close/main.jsx

```diff
@@ -73,6 +72,18 @@ class AccountSettingsClose extends Component {
 		this.setState( { showConfirmDialog: false } );
 	};
 
+	handleSiteDropdown = () => {
+		this.setState( {
+			showSiteDropdown: true,
+		} );
+
+		if ( this.state.showSiteDropdown ) {
+			this.setState( {
+				showSiteDropdown: false,
+			} );
+		}
+	};
```

Comment:
This code looks weird. If you want to toggle the state flag, use `this.setState( state => ( { showSiteDropdown: ! state.showSiteDropdown } ) )`

Reply:
Much more elegant! Thanks, done. :)